pages/windows/reg-compare.md

```diff
@@ -0,0 +1,27 @@
+# reg compare
+
+> Compare keys and their values in the registry.
+
+- Compare all values under a specific key with a second key:
+
+`reg compare {{first_key_name}} {{second_key_name}}`
+
+- Compare a specific value under two keys:
+
+`reg compare {{first_key_name}} {{second_key_name}} /v {{value}}`
+
+- Compare all sub keys and values for two keys:
+
+`reg compare {{first_key_name}} {{second_key_name}} /s`
+
+- Only output the differences between the specified keys (default):
```

Comment:
Is this example needed if it's the default operation mode, or could we remove it / merge it into the previous example?